test/specs.test.js

```diff
@@ -11,7 +11,9 @@ const clone = require('clone');
 
 const dir = path.resolve(__dirname, 'specs'),
     files = fs.readdirSync(dir).filter(function (f) {
-        return f.match(/\.yaml$/);
+        const isYaml = f.match(/\.yaml$/);
+        const grep = process.env.GREP ? (new RegExp(process.env.GREP)).test(f) : true;
```

Comment:
I was using this to be able to run:

`GREP="funtions*" npm test` to just run the function integration tests; just sped up my testing time; any thoughts about where I should document this?